packages/components/addon/components/hds/copy/snippet/index.hbs

```diff
@@ -0,0 +1,15 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<button
+  type="button"
+  class="{{this.classNames}} {{if this.isSuccess 'is-success'}}"
```

Comment:
Is there a reason to include the isSuccess logic in here vs. in the JS file? If there is, perhaps you could add a comment above the button explaining. Otherwise, I would move the logic to the JS to keep everything together.

Reply:
I would also use BEM style for this classname:

"is-success" => "hds-copy-snippet--is-success"

Reply:
oh, please ignore the copy/snippet for now, I wanted to see if the copy/button was closer to what we were thinking before refactoring the snippet. 👍